pyproject.toml

```diff
@@ -42,6 +42,9 @@ scripts.blurb = "blurb.blurb:main"
 [tool.hatch]
 version.source = "vcs"
 
+[tool.hatch.build.hooks.vcs]
+version-file = "src/blurb/_version.py"
```

Comment:
Do these two lines tell `hatch` to automatically create the `_version.py` file that contains `__version__ = '...'` on build?

Reply:
That's right: https://github.com/ofek/hatch-vcs#build-hook-options